user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,139 @@
+
+## How to register a private Catalog item
+
+### Register using the Catalog UI
+
+If you use the UI, you can only register a Catalog item. There is no UI yet to register a new version. This can only be done by using the API.
+
+To register a new Catalog item, do the following:
+
+1. Go to the [Catalog](https://catalog.dataminer.services).
+
+1. Open the user menu by clicking the user icon on the top right. Here you will find the "Register item" button.
+
+1. Indicate for which organization you want to register the Catalog item. This can be done by using the selection box that will automatically be populated with all the organizations on the dataminer.services platform you have access to.
+
+1. Select the type of the Catalog item. The type will define how the Catalog item will be shown to the users. We also allow filtering on type so users can quickly find what they are looking for when browsing the Catalog module.
+
+1. Enter the name of the item. This will be the name that will be used to display the item in the Catalog. It needs to be human readable. Make sure you pick a name that makes sense.
+
+1. Once all information has been filled in, click *Register*. This will conclude the registration process for the Catalog item and return the unique identifier for the item in question. This identifier will allow you to register versions for the catalog item using the [API](xref:Register_Catalog_Item#register-using-the-catalog-api).
+
+### Register using the Catalog API
+
+#### Authentication
+
+The API call is authenticated using [organization keys](xref:Managing_DCP_keys#organization-keys). The key needs to be added to the HTTP request in a header called **Ocp-Apim-Subscription-Key**.
+
+#### API call signature
+
+The call to register a Catalog item can be found at the following URL: <https://api.dataminer.serivces/api/key-catalog/v1-0/catalog/register>. To call the API you will need to use the **POST** HTTP method.
+
+The call accepts a binary file as input. This file should be a zip file containing a ["manifest.yml"](xref:Register_Catalog_Item#manifest-file) and the item you want to upload.
+
+#### Manifest file
+
+This file will contain all necessary information to register a Catalog item or version. This manifest file should be a valid yml file and will contain "required" and "optional" attributes to add extra information to the Catalog item. Note that limitations may apply to certain attributes based on length or formatting.
+
+```yml
+# [Required]
+# Possible values: 
+#   - AutomationScript: If the catalog item is an Automation Script that can be deployed on a DataMiner System.
+#   - AppPackage: If the catalog item is a Solution Package that can be deployed on a DataMiner System.
+#   - ProtocolPackage: If the catalog item is a Protocol that can be deployed on a DataMiner System.
+#   - Visio: If the catalog item is a Visio that can be deployed on a DataMiner System.
+type: 'AutomationScript'
```

Comment:
Did you check this with Data Acquisition? This is something that is typically also clear from the GitHub Action?

In this way, we can still make use of an Automation Script solution registering as a 'AppPackage' (ref. discussion earlier)? Can you elaborate?

Reply:
Yes this was checked by with Data Acquisition, they were in all meetings that were held to create this file.
The GitHub action will just pass this file to us.
It will indeed be possible to register an automation script as an app package if you would want to do so.
This type is not defining how something is packaged, it defines the type of the Catalog item as it will be shown on the catalog.